client/tests/integration/triggers/data_trigger.rs

```diff
@@ -41,72 +51,14 @@ fn must_execute_both_triggers() -> Result<()> {
     test_client.submit_blocking(Register::account(Account::new(
         gen_account_in("wonderland").0,
     )))?;
-    test_client.submit_blocking(Register::domain(Domain::new("neverland".parse()?)))?;
-
-    let new_value = get_asset_value(&test_client, asset_id);
-    assert_eq!(new_value, prev_value.checked_add(numeric!(2)).unwrap());
-
-    Ok(())
-}
-
-#[test]
-fn domain_scoped_trigger_must_be_executed_only_on_events_in_its_domain() -> Result<()> {
-    let (_rt, _peer, test_client) = <PeerBuilder>::new().with_port(10_655).start_with_runtime();
-    wait_for_genesis_committed(&[test_client.clone()], 0);
-
-    let create_neverland_domain: InstructionBox =
-        Register::domain(Domain::new("neverland".parse()?)).into();
-
-    let (account_id, _account_keypair) = gen_account_in("neverland");
-    let create_sapporo_account = Register::account(Account::new(account_id.clone())).into();
-
-    let asset_definition_id: AssetDefinitionId = "sakura#neverland".parse()?;
-    let create_sakura_asset_definition =
-        Register::asset_definition(AssetDefinition::numeric(asset_definition_id.clone())).into();
-
-    let asset_id = AssetId::new(asset_definition_id, account_id.clone());
-    let create_sakura_asset = Register::asset(Asset::new(asset_id.clone(), 0_u32)).into();
-
-    test_client.submit_all_blocking([
-        create_neverland_domain,
-        create_sapporo_account,
-        create_sakura_asset_definition,
-        create_sakura_asset,
-    ])?;
-
-    let prev_value = get_asset_value(&test_client, asset_id.clone());
-
-    let register_trigger = Register::trigger(Trigger::new(
-        "mint_sakura$neverland".parse()?,
-        Action::new(
-            [Mint::asset_numeric(1u32, asset_id.clone())],
-            Repeats::Indefinitely,
-            account_id,
-            AccountEventFilter::new().for_events(AccountEventSet::Created),
-        ),
-    ));
-    test_client.submit_blocking(register_trigger)?;
 
-    test_client.submit_blocking(Register::account(Account::new(
-        gen_account_in("wonderland").0,
-    )))?;
+    let new_value = get_asset_value(&test_client, asset_id.clone());
+    assert_eq!(new_value, prev_value.checked_add(numeric!(1)).unwrap());
 
-    test_client.submit_blocking(Register::account(Account::new(
-        gen_account_in("neverland").0,
-    )))?;
+    test_client.submit_blocking(Register::domain(Domain::new("neverland".parse()?)))?;
 
-    let new_value = get_asset_value(&test_client, asset_id);
-    assert_eq!(new_value, prev_value.checked_add(Numeric::ONE).unwrap());
+    let newer_value = get_asset_value(&test_client, asset_id);
+    assert_eq!(newer_value, new_value.checked_add(numeric!(1)).unwrap());
```

Comment:
Test is called `must_execute_both_triggers`, but looks like now only first trigger get executed. Is it expected?

Reply:
Both get executed, I check the decrement twice (`new_value` and `newer_value`).

Reply:
Yes, sorry